ui/apps/platform/src/Containers/NetworkGraph/components/NetworkSearch.css

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
It doesn't really impact this PR, but one thing I noticed with this is that we need to move the CSS to a more general location and/or import it in the other places it is used.

I believe the Violations and Policies pages are using this shim via classname as well since they are in PatternFly, but neither of those components import the CSS file. Since the CSS is lazy-loaded, if you directly visit one of those old pages you will get the old Tailwind styles, but then if you visit the Network Graph and return to the original page you will get the updated PT styles.

This has been on my personal todo list but I wanted to call it out since this CSS is moving to a new location.